chain/db.go

```diff
@@ -715,8 +860,8 @@ func (db *DBStore) PruneBlock(id types.BlockID) {
 func (db *DBStore) shouldFlush() bool {
 	// NOTE: these values were chosen empirically and should constitute a
 	// sensible default; if necessary, we can make them configurable
-	const flushSizeThreshold = 20e6
-	const flushDurationThreshold = 1000 * time.Millisecond
+	const flushSizeThreshold = 100e6
```

Comment:
`nit` Feels a bit high? For deletes we only count the key length so if the value is 1kb each this is roughly 3GB.

Reply:
right! woopsie